src/components/subscriptions/licenses/TabContentTable.jsx

```diff
@@ -56,8 +56,10 @@ const TabContentTable = ({ enterpriseSlug }) => {
     errors,
     setErrors,
   });
+
   const hasErrors = Object.values(errors).length > 0;
-  const hasLoadedUsers = !!(users?.numPages);
+  const hasLoadedUsers = !!(users?.numPages || users?.count);
```

Comment:
Note to self: file ticket for understanding why the endpoint in license-manager isn't returning these counts properly.